tests/cards/ares/DeimosDownAres.spec.ts

```diff
@@ -78,15 +80,13 @@ describe('DeimosDownAres', function() {
     const space = action.availableSpaces[0];
     action.cb(space);
 
-    expect(space.tile && space.tile.tileType).to.eq(TileType.DEIMOS_DOWN);
+    expect(space.tile?.tileType).to.eq(TileType.DEIMOS_DOWN);
     expect(space.adjacency).to.deep.eq({bonus: [SpaceBonus.ASTEROID, SpaceBonus.STEEL]});
   });
 
   it('Adjacency bonus - Tile Placement - Self', function() {
-    const adsPlayer1 = new AsteroidDeflectionSystem();
-    const adsPlayer2 = new AsteroidDeflectionSystem();
-    adsPlayer1.play(player);
-    adsPlayer2.play(player2);
+    player.playedCards.push(new AsteroidDeflectionSystem());
```

Comment:
`card.play` doesn't put it in the player's played card stack. It just executes the card.